.github/workflows/go.yml

```diff
@@ -36,6 +36,14 @@ jobs:
     - name: Run Golint
       run: make golint
 
+  verify-license:
```

Comment:
Can we have the license check in the lint job itself? And the build job should run only if the lint passes.

Reply:
change the order of action, 
- lint + license-check-->build

Reply:
Done